explorer/client/src/containers/JobRuns/Index.tsx

```diff
@@ -81,21 +96,23 @@ const jobRunsSelector = ({
   }
 }
 
-const jobRunsCountSelector = (state: IState) => {
+const jobRunsCountSelector = (state: State) => {
   return state.jobRunsIndex.count
 }
 
-const mapStateToProps = (state: IState) => ({
-  rowsPerPage: 10,
-  query: state.search.query,
-  jobRuns: jobRunsSelector(state),
-  count: jobRunsCountSelector(state)
-})
+const mapStateToProps = (state: State): StateProps => {
+  return {
+    rowsPerPage: 10,
+    query: state.search.query,
+    jobRuns: jobRunsSelector(state),
+    count: jobRunsCountSelector(state)
+  }
+}
 
-const mapDispatchToProps = (dispatch: Dispatch<any>) =>
+const mapDispatchToProps = (dispatch: Dispatch): DispatchProps =>
```

Comment:
There should be a `MapDispatchToProps` exported from `react-redux` that you can annotate this function with.